content/manuals/engine/install/sles.md

```diff
@@ -23,245 +21,36 @@ aliases:
-   `docker` group, however, it doesn't add any users to the group by default.
-  
-   {{< /tab >}}
-   {{< /tabs >}}
-
-2. Start Docker Engine.
-
-   ```console
-   $ sudo systemctl enable --now docker
-   ```
-
-   This configures the Docker systemd service to start automatically when you
-   boot your system. If you don't want Docker to start automatically, use `sudo
-   systemctl start docker` instead.
-
-3. Verify that the installation is successful by running the `hello-world` image:
-
-   ```console
-   $ sudo docker run hello-world
-   ```
-
-   This command downloads a test image and runs it in a container. When the
-   container runs, it prints a confirmation message and exits.
-
-You have now successfully installed and started Docker Engine.
-
-{{% include "root-errors.md" %}}
-
-#### Upgrade Docker Engine
-
-To upgrade Docker Engine, follow the [installation instructions](#install-using-the-repository),
-choosing the new version you want to install.
-
-### Install from a package
-
-If you can't use Docker's `rpm` repository to install Docker Engine, you can
-download the `.rpm` file for your release and install it manually. You need to
-download a new file each time you want to upgrade Docker Engine.
-
-<!-- markdownlint-disable-next-line -->
-1. Go to [{{% param "download-url-base" %}}/]({{% param "download-url-base" %}}/)
-   and choose your version of SLES. Then browse to `s390x/stable/Packages/`
-   and download the `.rpm` file for the Docker version you want to install.
-
-2. Install Docker Engine, changing the following path to the path where you downloaded
-   the Docker package.
-
-   ```console
-   $ sudo zypper install /path/to/package.rpm
-   ```
-
-   Docker is installed but not started. The `docker` group is created, but no
-   users are added to the group.
-
-3. Start Docker Engine.
-
-   ```console
-   $ sudo systemctl enable --now docker
-   ```
-
-   This configures the Docker systemd service to start automatically when you
-   boot your system. If you don't want Docker to start automatically, use `sudo
-   systemctl start docker` instead.
-
-4. Verify that the installation is successful by running the `hello-world` image:
-
-   ```console
-   $ sudo docker run hello-world
-   ```
-
-   This command downloads a test image and runs it in a container. When the
-   container runs, it prints a confirmation message and exits.
-
-You have now successfully installed and started Docker Engine.
-
-{{% include "root-errors.md" %}}
-
-#### Upgrade Docker Engine
-
-To upgrade Docker Engine, download the newer package files and repeat the
-[installation procedure](#install-from-a-package), using `zypper -y upgrade`
-instead of `zypper -y install`, and point to the new files.
-
-{{% include "install-script.md" %}}
+> [!IMPORTANT]
+>
+> Docker Engine packages are **no longer available** for SUSE Linux Enterprise Server (SLES) on the **s390x** architecture (IBM Z).
```

Comment:
It would be nice to add a date or version from which this change applies. That way we know when we can delete this section down the line when enough time has passed :)

Reply:
It's a bit unclear what's the exact "cut date" tbh. Basically v27.4.1 is the last published version for SLES s390x. But even the versions that are already published are no longer supported.